### PR TITLE
Updates to allow node count and ability to use environment var for -a

### PR DESCRIPTION
**Pull request by zachaller**
## Pull Request Checklist

**Is this in reference to an existing issue?**
#### General
- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
- [x] Update README with any necessary configuration snippets
- [x] Binstubs are created if needed
- [x] RuboCop passes
- [x] Existing tests pass 
#### New Plugins
- [ ] Tests
- [ ] Add the plugin to the README
- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by sstarcher** [comment]
Prefer SENSU_API_URL over SENSU_API to allow the user to set the port via the environment variable

**Comment by zachaller** [comment]
Ok um... so the reason for that was due to how we currently use it which maybe is not the best but we have a pretty big multi region deployment each having there own sensu server install, we would like just track the main project and not have to update all our configs yet but understand if you would want to keep the project generic. How do you feel about something like this

```
default: ENV['SENSU_API'] ? ENV['SENSU_API'] + ':4567' : ENV['SENSU_API_URL'] ? ENV['SENSU_API_URL'] : 'http://localhost:4567'
```

**Comment by sstarcher** [comment]
That seems "ok"
@sensu-plugins/commit-bit opinions?

**Comment by zachaller** [comment]
Now to just get it to pass I am not a ruby dev at all my first attempt returned 

```
bin/check-aggregate.rb:24:67: C: Ternary operators must not be nested. Prefer if or else constructs instead.
         default: ENV['SENSU_API'] ? ENV['SENSU_API'] + ':4567' : ENV['SENSU_API_URL'] ? ENV['SENSU_API_URL'] : 'http://localhost:4567'
```

so I switched it to if's but got this

```
bin/check-aggregate.rb:24:19: C: Favor the ternary operator (?:) over if/then/else/end constructs.
         default: if ENV['SENSU_API'] then ENV['SENSU_API'] + ':4567' elsif ENV['SENSU_API_URL'] then ENV['SENSU_API_URL'] else 'http://localhost:4567' end
```

**Comment by sstarcher** [comment]
@zachaller according to a google search I think the recommended way is

```
# bad
some_condition ? (nested_condition ? nested_something : nested_something_else) : something_else

# good
if some_condition
  nested_condition ? nested_something : nested_something_else
else
  something_else
end
```

**Comment by zachaller** [comment]
Well i tried to keep it on a single line and failed what i ended up doing works and passes and looks fairly clean i think also.

File: bin/check-aggregate.rb [hunk 1]
```diff
@@ -21,8 +21,14 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          short: '-a URL',
          long: '--api URL',
          description: 'Sensu API URL',
-         default: 'http://localhost:4567'
-
+         default:
+           if ENV['SENSU_API']
```
Comment (sstarcher) [inline bin/check-aggregate.rb]: would it make sense for this if line to be on the same line as the `default:`

something like

```
    default: if ENV['SENSU_API']
                   blah
                 else
                    blah
                 end
```

**Comment by zachaller** [comment]
Yea I got that to pass I had tried something like that before but did not indent the rest as far so it was throwing errors from rubocop but got it to pass the way you did it.

**Comment by sstarcher** [comment]
@zachaller thanks

**Pull request merged**